_topics/en/offers.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Offers
+
+## Optional.  Shorter name to use for reference style links e.g., "foo"
+## will allow using the link [topic foo][].  Not case sensitive
+# shortname: foo
+
+## Optional.  An entry will be added to the topics index for each alias
+#aliases:
+#  - Foo
+
+## Required.  At least one category to which this topic belongs.  See
+## schema for options
+categories:
+  - Lightning Network
+
+## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
+## Should be less than 500 characters
+excerpt: >
+  **Offers** is a proposed protocol enhancement for Lightning that would
+  allow nodes to request and receive invoices over LN.
+
+## Optional.  Use Markdown formatting.  Multiple paragraphs.  Links allowed.
+extended_summary: |
+  An example of a common use of this protocol would be that a merchant
+  generates a QR code, the customer scans the QR code, the customer’s LN
+  node sends some of the details from the QR code (such as an order ID
+  number) to the merchant’s node over LN, the merchant’s node returns an
+  invoice (also over LN), the invoice is displayed to the user (who
+  agrees to pay), and the payment is sent.
+
+  Although the above use case is already addressed today using
+  [BOLT11][] invoices, the ability for the spending and receiving nodes
+  to communicate directly before attempting payment provides much more
+  flexibility. It also enables features that aren’t possible with
```

Comment:
I think the flexibility could be elaborated upon with an example or noting some aspects that are more flexible.